### PR TITLE
run phpstorm base style formatter

### PR DESCRIPTION
**Pull request by bradfordcondon**
# Style Fix

Issue #310

## Description

This PR runs the PHP storm drupal style standard. This is less intense than the PHPCS drupal standard.

what gets changed:

* `array()` -> []
* white space 
* line length
* weird nested if elseifs
* other stuff...
  
screen shot 2019-03-05 at 8 09 47 pm

### CODE REVIEW

File: tripal_chado/includes/tripal_chado.fields.inc [hunk 1]
```diff
@@ -208,6 +212,9 @@ function tripal_chado_bundle_fields_info_base(&$info, $details, $entity_type, $b
       $base_info['type'] = 'text';
       $base_info['settings']['text_processing'] = 0;
     }
+
+    dpm($base_info);
```
Comment (laceysanderson) [inline tripal_chado/includes/tripal_chado.fields.inc]: How did this get added? Was testing and saw dpms when finding new fields 🤣 

```suggestion
```

Comment (laceysanderson) [reply tripal_chado/includes/tripal_chado.fields.inc]: Confirmed this is the only one that was introduced.

**Review by laceysanderson — COMMENTED** [review]
Other then the introduced dpm, everything else looks good. I checked a number of pages, cleared the cache 3X, looked for new fields, created content types, etc. (overkill really 😝 ) and everything worked the same way it did before

File: tripal_chado/includes/tripal_chado.fields.inc (hunk 1 above)
Comment (bradfordcondon) [reply tripal_chado/includes/tripal_chado.fields.inc]: ack!  I was troubleshooting issue #868 , thats where this DPM came from.
 Thank you for confirming this was the only one.

**Review by laceysanderson — APPROVED** [review]
I checked locally and all seems good 👍 Ignoring the failed test since the log implies it's the pubmed test being unable to query Entrez

**Comment by spficklin** [comment]
I did not do as thorough a review as Lacey. Just checked for a few major admin pages and it seems good. Nice to have the reformatting done!

**Review by spficklin — APPROVED** [review]

**Pull request merged**